CHANGELOG.md

```diff
@@ -10,7 +10,8 @@
 
 - Fall back to version 2 project config if version 3 fails. ([#1314](https://github.com/getsentry/relay/pull/1314))
 - Reduce number of metrics extracted for release health. ([#1316](https://github.com/getsentry/relay/pull/1316))
-- Indicate with thread is the main thread in thread metadata for profiles. ([#1320](https://github.com/getsentry/relay/pull/1320))
+- Extend trace sampling protocol to deal with flat user data. ([#1318](https://github.com/getsentry/relay/pull/1318))
```

Comment:
I think this is a feature.

Reply:
I made it internal because it shouldn't even be user-visible that we made this change, since we effectively just renamed a bunch of fields.